packages/terra-form-input/src/Input.module.scss

```diff
@@ -26,6 +26,15 @@
       color: var(--terra-form-input-placeholder-color, #70787c);
       font-style: var(--terra-form-input-placeholder-font-style);
     }
+    
+    /* TODO: Remove vendor prefix once we update to autoprefixer v9 */
```

Comment:
I'd document this on the issue logged to update AutoPrefixer.

Reply:
Can we link a github issue to reference?

Reply:
added comment in issue logged for dependency updates in terra-toolkit : https://github.com/cerner/terra-toolkit/issues/161#issuecomment-485660417

Reply:
@emilyrohrbough do you mean adding reference of issue logged for dependency update or the form-input issue for which I have added this code changes.

Reply:
@supreethmr Lets add it here so later on we will know this was a file that needs updated

Reply:
done : https://github.com/cerner/terra-core/blob/88ec784c6421f99916bf8abcc9723556f18b0bbb/packages/terra-form-input/src/Input.module.scss#L31